rdl/rdl-gen-athenz-java-client/javaclient.go

```diff
@@ -253,6 +253,8 @@ public class {{cName}}Client {
             BasicClientCookie cookie = new BasicClientCookie(header.substring(7), token);
             cookie.setPath(baseUrl);
             cookieStore.addCookie(cookie);
+            cookie.setHttpOnly(true);
+            cookie.setSecure(true);
```

Comment:
minor nitpick - but can we please move those 2 lines above the addCookie line just to get the order as expected.

Reply:
updated